ast/decl_stmt.c2

```diff
@@ -18,35 +18,52 @@ module ast;
 import ast_context;
 import string_buffer;
 
+import string local;
+
+type DeclStmtBits struct {
+    u32 : NumStmtBits;
+    u32 count : 5;
+}
+
 public type DeclStmt struct @(opaque) {
     Stmt base;
-    VarDecl* decl;
+    VarDecl*[0] decl;
 }
 
 public fn DeclStmt* DeclStmt.create(ast_context.Context* c,
-                                      VarDecl* decl)
+                                    VarDecl** decl, u32 count)
 {
-    DeclStmt* s = c.alloc(sizeof(DeclStmt));
-    s.base.init(StmtKind.Decl, decl.asDecl().getLoc());
-    s.decl = decl;
+    DeclStmt* s = c.alloc(sizeof(DeclStmt) * sizeof(VarDecl*) * count);
+    s.base.init(StmtKind.Decl, decl[0].asDecl().getLoc());
```

Comment:
The alloc seems a bit much.... a * b * c, probably a + b * c  ?

Reply:
Good catch!  updated the PR and fixed the stats too.